nym-api/src/support/config/template.rs

```diff
@@ -17,6 +15,9 @@ id = '{{ base.id }}'
 # Validator server to which the API will be getting information about the network.
 local_validator = '{{ base.local_validator }}'
 
+# Socket address this api will use for binding its http API.
```

Comment:
perhaps add a comment that this will only apply when `axum` feature flag is used

Reply:
done